### ISSUE TITLE
no command line tool in 20161201.0.0?

### ISSUE THREAD

**Issue by jiyinyiyong** [google/closure-compiler-js#51]
As I installed from npm I see no command line provided, but seeing from code in `master` branch, it does. And a `cmd.js` is missing:

```bash
=>> l /usr/local/lib/node_modules/google-closure-compiler-js
CONTRIBUTING.md  README.md        compile.js       index.js         lib/             node_modules/
LICENSE          build.js         contrib/         jscomp.js        logger.js        package.json
```

**Comment by MatrixFrog** [comment]
Looks like it was added in https://github.com/google/closure-compiler-js/pull/45 cc @samthor

**Comment by samthor** [comment]
We've released a new version, 20161201.0.1 that includes the command-line script.
